### ISSUE TITLE
Award xp window doesn't display checkbox

### ISSUE THREAD

**Issue by drdwing** [death-save/combat-utility-belt#559]
**Describe the bug**
When End combat window title text is changed, it doesn't display "Award XP?" checkbox.

**To Reproduce**
1. Change "COMBAT.EndTitle" of core.
2. Click combat end button.

**Screenshots**

![Cap 2021-06-18 19-55-13-837](https://user-images.githubusercontent.com/66671688/122551742-5d96a380-d070-11eb-85dd-aa7a426520ae.png)
![Cap 2021-06-18 19-42-53-919](https://user-images.githubusercontent.com/66671688/122551751-612a2a80-d070-11eb-9595-e465bbaebaf4.jpg)
![Cap 2021-06-18 19-41-19-303](https://user-images.githubusercontent.com/66671688/122551753-625b5780-d070-11eb-9624-f8bfbea3f5eb.jpg)

**Environment Info (please complete the following information):**
 - Foundry Version: 0.8.7
 - Game System / Version: dnd5e 1.3.5
 - CUB Version 1.6.0

**Comment by cbouyer** [comment]
Hello. This dialog's title is hardcoded in modules/signal.js file (row 197). Here is a quick fix:
```javascript
Hooks.on("renderDialog", (app, html, data) => {
    if (app.title === game.i18n.localize("COMBAT.EndTitle")) {
        GiveXP._onRenderDialog(app, html, data);
    }
});
```

**Comment by drdwing** [comment]
Thank you!! 👍👍

**Comment by eclarke12** [comment]
> Hello. This dialog's title is hardcoded in modules/signal.js file (row 197). Here is a quick fix:
> 
> ```js
> Hooks.on("renderDialog", (app, html, data) => {
>     if (app.title === game.i18n.localize("COMBAT.EndTitle")) {
>         GiveXP._onRenderDialog(app, html, data);
>     }
> });
> ```

do you want to put in a PR so I can credit you in the git blame?

**Comment by Daimakaicho** [comment]
> > Hello. This dialog's title is hardcoded in modules/signal.js file (row 197). Here is a quick fix:
> > ```js
> > Hooks.on("renderDialog", (app, html, data) => {
> >     if (app.title === game.i18n.localize("COMBAT.EndTitle")) {
> >         GiveXP._onRenderDialog(app, html, data);
> >     }
> > });
> > ```
> 
> do you want to put in a PR so I can credit you in the git blame?

Done #578

**Comment by eclarke12** [comment]
Resolved in #584
